### PR TITLE
fix: openapi3 requestBody work around support

### PR DESCRIPTION
**Pull request by eddielischk01**

### CODE REVIEW

File: lib/__tests__/expressMiddleware.test.js [hunk 1]
```diff
@@ -37,12 +46,51 @@ describe("Test wrapResponse", () => {
     res.write(Buffer.from("www"))
     res.write(Buffer.from("www"))
```
Comment (davidnghk01) [inline lib/__tests__/expressMiddleware.test.js]: May need one more test while `write` is `String` instance

Comment (eddielischk01) [reply lib/__tests__/expressMiddleware.test.js]: this include in the other PR
https://github.com/davidNHK/node-swagger-middleware/pull/5

File: lib/expressMiddleware.js [hunk 2]
```diff
@@ -133,11 +143,28 @@ function validateResponse(data, res, routeSpec) {
...
+      `request:${schema.ptr}`,
+      utils.deepFindByKey("schema")(schema.requestBody)
+    )
+
+    const validate = ajvCompiler.compile(jsonSchema)
+    const isJsonDataMatchSchema = validate(data)
+    if (!isJsonDataMatchSchema) {
+      throw new AJVValidationError(validate.errors)
+    }
+    return Buffer.from(JSON.stringify(data))
```
Comment (davidnghk01) [inline lib/expressMiddleware.js]: Why?

Comment (eddielischk01) [reply lib/expressMiddleware.js]: copy wrong.....

File: lib/expressMiddleware.js [hunk 3]
```diff
@@ -155,7 +155,6 @@ function validateRequestByAJV(data, schema, ajvCompiler) {
     if (!isJsonDataMatchSchema) {
       throw new AJVValidationError(validate.errors)
     }
-    return Buffer.from(JSON.stringify(data))
   }
   return undefined
```
Comment (davidnghk01) [inline lib/expressMiddleware.js]: return undefined is not necessary

**Pull request merged**